ods_ci/utils/scripts/rosa/rosaOps.py

```diff
@@ -0,0 +1,97 @@
+from util import (
+    execute_command,
+)
+from logging import log
+from time import sleep
+import sys
+
+def create_account_roles():
+    cmd_create_account_roles=["rosa", "create", "account-roles", "--mode", "auto", "--yes"]
+
+    
+    print(' '.join(cmd_create_account_roles))
+    ret = execute_command(' '.join(cmd_create_account_roles))
+    if ret is None:
+        print("Failed  to Create account roles")
+        return ret       
+
+def rosa_create_cluster(cluster_name, region ,channel_name, compute_nodes,compute_machine_type,rosa_version):
+    cmd_rosa_create_cluster = ["rosa", "create", "cluster", "--cluster-name", cluster_name, "--replicas   ", compute_nodes, "--region", region, "--compute-machine-type", compute_machine_type, "--yes", "--sts", "--version", rosa_version, "--channel-group", channel_name]
+    print(' '.join(cmd_rosa_create_cluster))
+    execute_command(' '.join(cmd_rosa_create_cluster))
+
+
+    
+    cmd_create_operator_roles= ["rosa", "create", "operator-roles", "--cluster", cluster_name, "--mode", "auto", "--region", region, "--yes"]
+    ret = execute_command(' '.join(cmd_create_operator_roles))
+    print(' '.join(cmd_create_operator_roles))
+    if ret is None:
+        print("Failed  to Create operator-roles")
+        return ret
+
+    cmd_create_oidc_provider = ["rosa", "create", "oidc-provider", "--cluster", cluster_name, "--mode", "auto", "--region", region, "--yes"]
+    ret = execute_command(' '.join(cmd_create_oidc_provider)) 
+    print(''.join(cmd_create_oidc_provider))
+    if ret is None:
+        print("Failed  to Create oidc roles")
+        return ret
+
+    cmd_check_cluster = ["rosa", "describe", "cluster", "--cluster={}".format(cluster_name)] #, '|', 'wc', '-l']
+    ret = execute_command(' '.join(cmd_check_cluster))
+    if ret is None:
+        print("Failed  creation failed")
+        return ret
+    print("ret = {}".format(ret))
+
+
+def rosa_describe(cluster_name, filter=""):
+    """Describes cluster and returns cluster info"""
+    cmd = "rosa describe cluster --cluster {}".format(cluster_name)
+    if filter != "":
+        cmd += " " + filter
+    ret = execute_command(cmd)
+    if ret is None:
+        print("rosa describe for cluster " "{} failed".format(cluster_name))
+        return None
+    return ret
+
+def get_osd_cluster_state(cluster_name):
+    """Gets osd cluster state"""
+
+    cluster_state = rosa_describe(cluster_name, filter="--output json | jq -r '.state'")
+    if cluster_state is None:
+        print(
+            "Unable to retrieve cluster state for "
+            "cluster name {}. EXITING".format(cluster_name)
+        )
+        sys.exit(1)
+    return cluster_state.strip("\n")
+    
+
+def wait_for_osd_cluster_to_be_ready(cluster_name, timeout=7200):
```

Comment:
This is quite confusing naming, pls consider using something like `wait_for_rosa_cluster_to_be_ready`